megazords/ios-rust/generate-files.sh

```diff
@@ -25,6 +25,12 @@ CARGO="$HOME/.cargo/bin/cargo"
 "$CARGO" uniffi-bindgen-library-mode -l "$UNIFFI_BINDGEN_LIBRARY" swift --headers "$COMMON/Headers"
 "$CARGO" uniffi-bindgen-library-mode -l "$UNIFFI_BINDGEN_LIBRARY" swift --modulemap "$COMMON/Modules" --xcframework --modulemap-filename module.modulemap
 
+## Tests will need the generated swift files from uniffi
+# TODO: Should we wrap this around an argument? we'd only need this for tests
```

Comment:
What other processes need to use this script? Is it (just) rust-components-swift? It might be worth conditionally executing this script only when needed so that the processes that don't require it can skip it and bypass any future bustages.

Reply:
generate-files is mainly just for building the xcframework, so in actual processes it would take a couple of actual seconds but do nothing with those sources. However I totally forgot to add a nice wrapper to ensure we only do it if we pass something like `--generate-sources`, which I did now -- thanks for catching that!